### DISCUSSION TITLE
Package 'openjdk-11-jre' has no installation candidate

### DISCUSSION THREAD

**Discussion by abdullahodibat** [apache/airflow#31938] [General] [answered]
Hello Everyone,
I am using airflow through docker container, deployed on AWS fargate as a service which has been working completely fine for a long time but suddenly today I'm unable to deploy any new changes to airflow and DAGs with the following error: 
<img width="1049" alt="Screenshot 2023-06-15 at 16 22 10" src="https://github.com/apache/airflow/assets/15142264/d824fb94-689f-4d7c-b10e-b1de85b8566d">

This is my docker file:
```
FROM apache/airflow:2.5.0-python3.9

ENV AIRFLOW_HOME=/usr/local/airflow

USER root

EXPOSE 8080

RUN apt-get update && apt-get install -y python3-pip \
    libcurl4-gnutls-dev \
    librtmp-dev \
    python3-dev \
    libpq-dev \
    unzip

RUN curl https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip -o awscliv2.zip \
    && unzip awscliv2.zip \
    && ./aws/install \
    && rm awscliv2.zip \
    && rm -rf ./aws

RUN mkdir -p ${AIRFLOW_HOME} && chown -R airflow: ${AIRFLOW_HOME}

USER airflow

RUN python -m pip install -U pip

COPY requirements.txt ${AIRFLOW_HOME}/requirements.txt
RUN python -m pip install -r ${AIRFLOW_HOME}/requirements.txt

COPY ./scripts/* ${AIRFLOW_HOME}/
COPY ./config ${AIRFLOW_HOME}/config
COPY ./plugins ${AIRFLOW_HOME}/plugins

WORKDIR ${AIRFLOW_HOME}
```
Any one faced this error before?

Thanks in advance!

**Comment by potiuk** [comment]
Nothing on the side of Airflow - the image has not changed for 6 months. It must have been debian, removing openjdk-11 from their repository - which would not be strange because openjdk-11 reaches end of life in 3 months. 

I'd recommend to review what you install and make sure to use a modern JDK for your installation

**Reply by potiuk** [reply, marked as answer]
BTW. you crucially omitted the `openjdk-11`  from your Dockerfile but in the output it is clearly visible that you are trying to install it, so maybe you posted a wrong Dockerfile.

**Reply by abdullahodibat** [reply]
@potiuk Thanks for pointing out! there was another docker file in the repo that contained the `openjdk-11`,,, i will update it to download version `17` instead!

Thanks for your help :)

**Reply by potiuk** [reply]
Ah I know. You are using bookworm, released last week in the other image (maybe accidentally) https://www.debian.org/releases/bookworm/. Airflow still uses Bullseye LTS. I guess the new Bookworm release does not have openjdk-11 any more.
